docs/javascript-installation.md

```diff
@@ -0,0 +1,115 @@
+---
+id: javascript-installation
+title: Scope Javascript Agent installation
+sidebar_label: Installation
+---
+
+Installation of the Scope Agent is done via [npm](https://www.npmjs.com/package/@undefinedlabs/scope-agent).
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
+
+```bash
+npm install --save-dev @undefinedlabs/scope-agent
+```
+
+<!--yarn-->
+
+```bash
+yarn add --dev @undefinedlabs/scope-agent
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Instrumenting your tests
+
+### Jest tests
+
+If you want to instrument tests run by Jest, you need to create a `reporter`.
+
+In `src/reporter.js`
+
+```javascript
+const Reporter = require("@undefinedlabs/scope-agent").JestReporter;
+
+module.exports = Reporter;
+```
+
+Then you have to configure [jest reporter](https://jestjs.io/docs/en/configuration#reporters-array-modulename-modulename-options)
+to point to this file.
+
+For example:
+
+In your `jest.config.js`:
+
+```javascript
+module.exports = {
+  // ...
+  reporters: ["<rootDir>/reporter.js"]
+  // ...
+};
+```
+
+#### Note for Create React App users
```

Comment:
I'd say `Create React App` and give the command line first, a then the explanation. I don't care if CRA allows or not reporter config in the package.json